src/ResourceGraph/ResourceGraph/Az.ResourceGraph.psd1

```diff
@@ -70,7 +70,7 @@ FormatsToProcess = 'ResourceGraph.format.ps1xml',
                'ResourceGraph.Autorest\Az.ResourceGraph.format.ps1xml'
 
 # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
-NestedModules = @('Az.ResourceGraph.psm1', 
+NestedModules = @('Microsoft.Azure.PowerShell.Cmdlets.ResourceGraph.dll', 
```

Comment:
Please let me know if I should revert this change. I wasn't able to run tests locally without it.